operate/importer/src/main/java/io/camunda/operate/zeebeimport/elasticsearch/ElasticsearchRecordsReader.java

```diff
@@ -624,6 +624,12 @@ private boolean isPartitionCompletedImporting(final int partitionId) {
           .filter(Objects::nonNull)
           .map(Object::toString)
           .anyMatch(indexName -> indexName.contains("8.8"));
+    } catch (final ElasticsearchStatusException e) {
+      LOGGER.warn(
+          "Elasticsearch error when checking if partition {} completed importing. Will retry later: {}",
```

Comment:
Omit the part about retrying later, this is logic handled by the calling method and is not a good fit in this catch block